test-radar.sh

```
@@ -282,6 +282,14 @@ validate_zip() {
 
 	echo "  PostgreSQL: $pg_count, System: $sys_count, pg_statviz: $statviz_count, Empty files: $empty_count"
 
+	# The archive is owner-only: it holds pg_hba.conf, connection details and
+	# query text, and is written into a directory other local users can read.
+	local mode=$(stat -c %a "$zip_file")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)